.spin/cmds.py

```diff
@@ -263,6 +263,38 @@ def _run_asv(cmd):
 
     util.run(cmd, cwd='benchmarks', env=env)
 
+@click.command()
+@click.option(
+    "-b", "--branch",
+    metavar='branch',
+    default="main",
+)
+@click.option(
+    '--uncommitted',
+    is_flag=True,
+    default=False,
+    required=False,
+)
+@click.pass_context
+def lint(ctx, branch, uncommitted):
+    """🔦 Run lint checks on diff.
+    Provide target branch name or `uncommitted` to check before committing:
+
+    \b
+    Examples:
+
+    \b
+    $ spin lint
+    $ spin lint --branch main
+    $ spin lint uncommitted
+    """
+    click.secho(
+        f"Performing lint checks against {'uncommitted changes' if uncommitted else branch + ' branch'}",
```

Comment:
This isn't right I think - it should be the current state of the repo, including uncommitted changes. The point of the flag is to ignore anything but uncommitted changes (useful in case there's a known problem elsewhere I think).

Reply:
Oh right, I meant that. Will fix it 👍